nni/algorithms/compression/pytorch/pruning/sensitivity_pruner.py

```diff
@@ -146,18 +146,23 @@ def validate_config(self, model, config_list):
 
         if self.base_algo == 'level':
             schema = CompressorSchema([{
-                'sparsity': And(float, lambda n: 0 < n < 1),
+                Optional('sparsity'): And(float, lambda n: 0 < n < 1),
                 Optional('op_types'): [str],
                 Optional('op_names'): [str],
+                Optional('exclude'): bool
             }], model, _logger)
         elif self.base_algo in ['l1', 'l2', 'fpgm']:
             schema = CompressorSchema([{
-                'sparsity': And(float, lambda n: 0 < n < 1),
+                Optional('sparsity'): And(float, lambda n: 0 < n < 1),
                 'op_types': ['Conv2d'],
-                Optional('op_names'): [str]
+                Optional('op_names'): [str],
+                Optional('exclude'): bool
             }], model, _logger)
 
         schema.validate(config_list)
+        for config in config_list:
+            if 'exclude' not in config and 'sparsity' not in config:
```

Comment:
Only when both `sparsity` and `op_types/op_names` exists, this config is validate, right?

Reply:
yes, and you reminded me this way is not elegant, modify it.